CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
```

Comment:
Do we _really_ need this?

Reply:
No, but it's one of the standard suggestions and I don't see a reason to not have it.